src/database/repository/diesel/item.rs

```diff
@@ -0,0 +1,43 @@
+use crate::database::repository::repository::get_connection;
+use crate::database::repository::RepositoryError;
+use crate::database::schema::ItemRow;
+
+use diesel::prelude::*;
+use diesel::r2d2::ConnectionManager;
+use r2d2::Pool;
+
+use super::DBBackendConnection;
```

Comment:
Minor formatting nits. Some general rust conventions, some more arbitrary and specific to this project - hopefully nothing too controversial!

1. Squash imports which have a shared root crate. 
2. Avoid importing same crate from different namespaces (e.g. `r2d2` vs. `diesel::r2s2`).
3. Local imports grouped at the top of the file, external crates imported afterwards (separated by a newline).

E.g.

```suggestion
use super::DBBackendConnection;
use crate::database::{
    repository::{repository::get_connection, RepositoryError},
    schema::ItemRow,
};

use diesel::{
    prelude::*,
    r2d2::{ConnectionManager, Pool},
};
```

Reply:
Any automated tooling for it?

Reply:
Not at the moment. Could update `rustfmt` to help out with some of it, if you agree with it?